### PR TITLE
Issue 43: Add noconfirm option

### PR DESCRIPTION
**Pull request by NgoHuy**
I just wrote some code to fix issue #43 
- Fix Rns logic for dpkg
- Add --noconfirm support
- Correct function name: _translate_w
- Add yum and dpkg's option --noconfirm
- Correct regex to check yum-*-downloadonly installed.

### CODE REVIEW

**Comment by icy** [comment]
Thank you Huy I'll take a look at weekend.

File: lib/00_core.sh [hunk 1]
```diff
@@ -95,8 +95,8 @@ _tranlate_w() {
 
   "yum")
     _TOPT="--downloadonly"
-    if ! rpm -q 'yum-downloadonly' >/dev/null 2>&1; then
-      _error "'yum-downloadonly' package is required when '-w' is used."
+    if ! [[ $(rpm -qa --pipe "grep downloadonly") ]]; then
```
Comment (icy) [inline lib/00_core.sh:98]: The output of `rpm` should not be used as an expression.

File: lib/zz_main.sh [hunk 2]
```diff
@@ -74,14 +80,14 @@ while :; do
...
-    s|l|i|p|o|m|n)
+    s|l|i|p|o|m|n|g)
       if [[ "$_SOPT" == '' ]]; then
         _SOPT="$_opt"
       else
         if [[ "${_SOPT:0:1}" == "s" ]]; then
           _SOPT="ns"
         else
-          _SOPT="n"
+          _SOPT="$_SOPT$_opt"
```
Comment (icy) [inline lib/zz_main.sh:90]: The correct fix for this is https://github.com/icy/pacapt/commit/99ba2c14906bf8335435834d3d032acfbc0b7d94

File: lib/00_core.sh (hunk 1 above)
Comment (NgoHuy) [inline lib/00_core.sh:98]: I tested with redirect and saw it failed with nothing print, I have no idea now.

File: lib/zz_main.sh (hunk 2 above)
Comment (NgoHuy) [inline lib/zz_main.sh:90]: Thank you for better fix,

**Comment by NgoHuy** [comment]
Because conflict in your new commit, I will delete this pull and uptodate with you, thank you very much for reviewing.

**Comment by icy** [comment]
I will try to cherry pick some good stuff from the Pr. 

Next time don't put a mess in a single PR. Thanks :)

**Comment by NgoHuy** [comment]
I noted, thank you very much

**Pull request closed without merging**